pilot/pkg/kube/inject/app_probe_test.go

```diff
@@ -43,3 +43,24 @@ func TestFindSidecar(t *testing.T) {
 		}
 	}
 }
+
+func TestShouldRewriteAppProbers(t *testing.T) {
+	for _, tc := range []struct {
+		name                 string
+		sidecarInjectionSpec SidecarInjectionSpec
+		annotations          map[string]string
+		expected             bool
+	}{
+		{"RewriteAppHTTPProbe-not-set", SidecarInjectionSpec{RewriteAppHTTPProbe: false}, map[string]string{}, false},
```

Comment:
can you break struct into several lines for better readability? currently this struct is a bit long and hard to follow.

Reply:
changed. please have a look